frontend/src/lib/components/CommandBar/SearchResults.tsx

```diff
@@ -0,0 +1,60 @@
+import { useActions, useValues } from 'kea'
+import { useEventListener } from 'lib/hooks/useEventListener'
+import { DetectiveHog } from '../hedgehogs'
+
+import { searchBarLogic } from './searchBarLogic'
+import SearchResult, { SearchResultSkeleton } from './SearchResult'
+
+const SearchResults = (): JSX.Element => {
+    const { filterSearchResults, searchResponseLoading, activeResultIndex, keyboardResultIndex, maxIndex } =
+        useValues(searchBarLogic)
+    const { onArrowUp, onArrowDown, openResult } = useActions(searchBarLogic)
+
+    useEventListener('keydown', (event) => {
+        if (!filterSearchResults) {
+            return
+        }
+
+        if (event.key === 'Enter') {
+            event.preventDefault()
+            openResult(activeResultIndex)
+        } else if (event.key === 'ArrowDown') {
+            event.preventDefault()
+            onArrowDown(activeResultIndex, maxIndex)
+        } else if (event.key === 'ArrowUp') {
+            event.preventDefault()
+            onArrowUp(activeResultIndex, maxIndex)
+        }
+    })
+
+    return (
+        <div className="grow overscroll-none overflow-y-auto">
+            {searchResponseLoading && (
```

Comment:
There is a weird flickering when searching without results
![flickering](https://github.com/PostHog/posthog/assets/6685876/3938c2e8-1036-4498-bf93-7d80164cc865)

Reply:
This is switching between the loading and empty state. I imagine it might feel ok on the live version, when the api requests actually take a bit of time to come back. Will keep an eye on this and a obvious solution would be to stop making futher request when a shorter search came back without results.

Reply:
Sounds good 👍 One thing that came to mind - is this debouncing at all? We should probably include it if not so that we don't hammer the API

Reply:
It's not, but we totally should. Will add a debounce.